src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -150,25 +150,36 @@ object BlockMatrix {
 
     implicit class Shim(l: M) {
       def +(r: M): M = l.add(r)
+
       def -(r: M): M = l.sub(r)
+
       def *(r: M): M = l.mul(r)
+
       def /(r: M): M = l.div(r)
-      
+
       def +(r: Double): M = l.scalarAdd(r)
+
       def -(r: Double): M = l.scalarSub(r)
+
       def *(r: Double): M = l.scalarMul(r)
+
       def /(r: Double): M = l.scalarDiv(r)
 
       def T: M = l.transpose()
     }
 
     implicit class ScalarShim(l: Double) {
       def +(r: M): M = r.scalarAdd(l)
+
       def -(r: M): M = r.reverseScalarSub(l)
+
       def *(r: M): M = r.scalarMul(l)
+
       def /(r: M): M = r.reverseScalarDiv(l)
     }
+
```

Comment:
Is this autoformat or manual? If the former, please relax change settings. If the latter, make your case. I prefer the grouping of parallel functions into tight groups.